cicd/bootstrap.sh

```diff
@@ -70,8 +71,8 @@ pip install --upgrade 'crc-bonfire>=4.10.4'
 
 # clone repo to download cicd scripts
 rm -fr $BONFIRE_ROOT
-echo "Fetching branch '$BONFIRE_BRANCH' of https://github.com/RedHatInsights/bonfire.git"
-git clone --branch $BONFIRE_BRANCH https://github.com/RedHatInsights/bonfire.git $BONFIRE_ROOT
+echo "Fetching branch '$BONFIRE_BRANCH' of https://github.com/${BONFIRE_GROUP}/bonfire.git"
```

Comment:
I just want to be picky about the names... maybe can we edit these variables to be called --
BONFIRE_REPO_BRANCH and BONFIRE_REPO_ORG

Reply:
Ack. Changed